src/release.js

```diff
@@ -94,7 +94,7 @@ module.exports = async function ({ github, context, inputs }) {
       await tagVersionInGit(`v${major}.${minor}`)
       await tagVersionInGit(`v${major}.${minor}.${patch}`)
     } else {
-      logError('missing sync-semver')
+      logWarning('missing sync-semver')
```

Comment:
why are we logging this in the first place? I mean if an input is not set, which is not mandatory, we shouldn't care really